### DISCUSSION TITLE
Questions on "Power Management" applet, options therein

### DISCUSSION THREAD

**Discussion by rajibando** [lxqt/lxqt#2605] [Q&A]
Hello, leaders and senior members of LXQt discussions github group.

Eventually, I decided to install LXQt from the _debian-live-12.6.0-amd64-lxqt.iso_ , on the 4th of September 2024, after using the Live ISO for a while. Appears to be a good decision.

The version of LXQt is ( StartMenu ⟶ About LXQt ) as follows:
  LXQt About Version:     1.2.0
  LXQt Version:           1.2.0
  Qt:                     5.15.8
The same info is also available from: 
$lxqt-session --version

May please the following applet be perused:
StartMenu ⟶ Preferences ⟶ "LXQt settings" ⟶ "Power Management"

My questions are:
Q1. Which aspects did the programmers of LXQt consider as Idleness?
Illustration: I set the option "Suspend" on the "Enable Idleness Watcher"  while I was listening to a long recording, while not touching the keyboard or mouse.
At the designated time, while the audio was still playing, the computer suspended itself.
Presently, I have temporarily opted for "Turn Off monitor(s)" until I have understood the minds of the programmers on this issue.

Q2. How do I set up the system so that if no audio/AV (like when I watch a film) is playing and I have not touched the keyboard or mouse, the computer would not suspend itself?

Q3.  Why the applet "Enable Backlight Change" doesn't work, though ACPI commands work, like "xset dpms force off" works?

The tab Idle, under "Power Management", has the following entries:
``------------------------``
Enable Idleness Watcher
Battery
When idle on AC:(Options)
`---------`
Nothing
Ask
Lock Screen
Suspend
Hibernate
Shutdown
Turn Off monitor(s)
`---------`
AC Idle time:
3:00

When idle on Battery:(Options)
`---------`
Nothing
Ask
Lock Screen
Suspend
Hibernate
Shutdown
Turn Off monitor(s)
`---------`

Battery Idle time:
02:00

✔ Enable Backlight Change
Backlight (%):
Idle time:
01:00
✔ On battery discharging
Check backlight
Disable idleness checks when fullscreen
Reset      × Close
`------------------------`

**Comment by tsujan** [comment]
> Which aspects did the programmers of LXQt consider as Idleness?

Like any other DE: key press and mouse movement.

> while the audio was still playing, the computer suspended itself.

Playing audio or video doesn't prevent idleness unless the playing app has a setting for imitating activity. They usually call it "disabling screensaver".

> Q2

Either uncheck Power Management Settings → Idle → Enable Idleness Watcher or choose "Nothing".

**Comment by stefonarch** [comment]
Q1: Idleness interrupt depends if the applications send some kind of "fake activity" afaik, so that depends on the applications.

Q2: fullscreen video should stop idleness control. There are also "suspend idleness checks" for different amounts of time (right click battery icon). I'm afraid this is an issue with PC without battery, I have a script to stop idleness checks for 1 hour which I trigger with an icon.

Q3: Backlight is for laptops, has no effect on monitors. There is "Brightness" in "Configuration Center" which fakes gamma.

**Comment by rajibando** [comment]
Thank you, Mr. Tsujan and Mr. stefonarch, for your prompt replies.

I have changed the option from "Suspend" to " Turn Off monitor(s)".

I also took a note of your points:
A1. ...  if the applications send some kind of "fake activity"  ... depends on the applications

A2. ... fullscreen video should stop idleness control ...

I have a two-part question:
Q1   (a) Are those listed options in the said applet "Power Management" in-built into the applet, or (b) can those options be changed by changing the contents of a list-of-values file?

If the case (b) is "Yes" then could a change be introduced to defer "suspend"-ing the laptop when audio is being played? Could playing an audio or an AV be considered as a mouse or keypress activity?

Mr. stefonarch, I am using the installation on my laptop. The screen of the laptop should ideally have both a software Brightness and a Contrast control.

Also, I consider the either of a Laptop and a Desktop a PC (personal computer). 
Q2.    Aren't the definitions we have for the terms on the two devices the same?

**_Observation:_**
The "Backlight" slider of the applet, StartMenu ⟶ Preferences ⟶ "LXQt settings" ⟶ "Brightness"  doesn't seem to work as intended. Instead, the EDP-1 works as "Brightness" control. There is no contrast control via 3-sliders controlling RGB.

On the contrary,  controlling Brightness and Contrast is much easier directly from the terminal.
Say, `cat /sys/class/backlight/intel_backlight/max_brightness`, to find the max brightness possible. Then controlling it.
Similarly, for contrast, `xrandr --output <display> --gamma <R>:<G>:<B>`
